### ISSUE TITLE
Code panel collapses files even if there are only a few files

### ISSUE THREAD

**Issue by damithc** [reposense/RepoSense#1772]
I noticed that the code panel collapses some files even when the number of files is small (e.g., there were only 4 files and 3 were collapsed). What's the threshold condition for collapsing the code view of files?

![image](https://user-images.githubusercontent.com/1673303/166906590-c43819e8-d2dc-47a2-8fa6-04e4dc3a3e6c.png)

Report in concern is [here](https://nus-cs2103-ay2122s2.github.io/ip-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=java~md~fxml~sh~bat~gradle~txt&since=2022-01-14&tabOpen=true&tabType=authorship&tabAuthor=Rdac0&tabRepo=Rdac0%2Fip%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=java~fxml~bat~gradle~txt&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)

**Comment by damithc** [comment]
@gok99 any idea about this one?

**Comment by gok99** [comment]
It seems like the overall character threshold set it in #1647, somewhat arbitrarily at 100k characters, is hit if both `DeveloperGuide.md` and `UserGuide.md` were to be un-collapsed. The overall character limit has a few associated issues and was largely meant to be a temporary solution until lazy-loading is fully realized in #1750. 

As a temporary fix, we could raise the overall character limit to a more reasonable value. In any case, #1750 will be getting rid of the overall character limit in place of a file-level limit which should resolve most of these issues.

Also, you may have meant to link the tp-dashboard instead: [here](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&since=2022-02-18&until=2022-04-22&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=Rdac0&tabRepo=AY2122S2-CS2103-W16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)

**Comment by damithc** [comment]
Thanks for the followup @gok99 
Yes, let's raise the limit for the time being, as it would be an easy fix (right?).

**Comment by gok99** [comment]
>Yes, let's raise the limit for the time being, as it would be an easy fix (right?).

Yes it should, after some playing around my computer seems to able to handle up to 3M character files without much trouble. I'll set the new limit at 3M.
